myproject/supermarket_manager/views.py

```diff
@@ -1,14 +1,15 @@
 from django.http import JsonResponse
 from django.db.utils import IntegrityError
 from django.views.decorators.csrf import csrf_exempt
-from .models import Role, Permission
+from .models import Role, Permission, Role_Permission
```

Comment:
Please import with the format: from .models import ( .... )

Reply:
Done